test/unit/Assessment/setup.js

```diff
@@ -65,8 +66,8 @@ async function setup() {
   const accounts = await getAccounts();
   await master.enrollGovernance(accounts.governanceContracts[0].address);
   for (const member of accounts.members) {
-    await master.enrollMember(member.address, 1); // Uses a different role value than IMemberRoles
-    await memberRoles.enrollMember(member.address, 2); // Uses the actual member role value
+    await master.enrollMember(member.address, Role.Member);
```

Comment:
This had a value of 1 (AB) previously, why the change?

Reply:
There were different values in the IMember and in MasterMock for the same role as it was in the comments before the fix.